docs/docs/reference/sandbox_reference/cli_reference.md

```diff
@@ -0,0 +1,378 @@
+---
+title: CLI Reference
+---
+
+:::warning
+
+`aztec-builder` and `aztec-sandbox` have been deprecated in favor of `aztec` CLI
+
+:::
+- [Start](#starting-and-testing)
+- [Accounts](#account-management)
+- [Contract deployments and interaction](#contract-deployment-and-interaction)
+- [Network and node info](#network-and-node-information)
+- [Querying](#transaction-and-block-querying)
+- [Logging](#logging-and-data-retrieval)
+- [Debugging](#development-and-debugging-tools)
+- [L1 contracts](#l1-contract-management)
+- [Utils](#utility-commands)
+
+## Starting and testing
+
+### start
+
+Initiates various Aztec modules. It can be used to start individual components or the entire Aztec Sandbox.
+
+```
+aztec start [options]
+```
+
+Options:
+- `-sb, --sandbox`: Starts the Aztec Sandbox.
+- `-p, --port <port>`: Specifies the port to run Aztec on (default: 8080).
+- `-n, --node [options]`: Starts the Aztec Node with specified options.
+- `-px, --pxe [options]`: Starts the PXE (Private eXecution Environment) with specified options.
+- `-a, --archiver [options]`: Starts the Archiver with specified options.
+- `-s, --sequencer [options]`: Starts the Sequencer with specified options.
+- `-r, --prover [options]`: Starts the Prover Agent with specified options.
+- `-o, --prover-node [options]`: Starts the Prover Node with specified options.
+- `-p2p, --p2p-bootstrap [options]`: Starts the P2P Bootstrap node with specified options.
+- `-t, --txe [options]`: Starts the TXE (Transaction Execution Environment) with specified options.
+
+### test
+
+Runs tests written in contracts.
+
+```
+aztec test [options]
+```
+
+Options:
+- `--silence-warnings`: Silences warnings during the test execution.
+- `--oracle-resolver <url>`: Sets the oracle resolver URL (default: "http://aztec:8081").
```

Comment:
```suggestion
```

Even though they're inherited from `nargo`, these two parameters will be overridden every time `aztec test` is executed to connect to the local txe instance (and not spam the user with warnings)